docs/music-server.md

```diff
@@ -361,3 +419,148 @@ DocumentRoot "C:\Bemuse\myserver"
 Once the file is saved, open your XAMPP Control Pannel and run "Apache".
 
 Then connect to the music server with (http://bemuse.ninja/?server=https://localhost/).
+
+#### macOS using [Nginx](https://nginx.org/en/)
+
+First, Open terminal and Use `cd` command, move to folder that bemuse custom files contained.
+
+Then, install Nginx
+
+```bash
+$ brew install nginx
+```
+
+<div class="admonition note">
+<p class="admonition-title">Note</p>
+<p>
+    Homebrew will install nginx on /usr/local.
+</p
```

Comment:
This is missing a `>`, causing layout breakage

![image](https://user-images.githubusercontent.com/193136/48305774-7b6c0580-e563-11e8-8bcc-f76feffdb342.png)

```suggestion
</p>
```

Reply:
Oh I missed that. Sorry, and I committed the fix. Thank you.